tests/steps/application_v2_steps.py

```diff
@@ -7,7 +7,11 @@
 
 from algosdk import abi, atomic_transaction_composer, encoding, mnemonic
 from algosdk.abi.contract import NetworkInfo
-from algosdk.error import ABITypeError, IndexerHTTPError
+from algosdk.error import (
```

Comment:
this diff should go away after `refactor-unit-tests` merges in the latest on `develop`